.github/workflows/reusable_ragger_tests.yml

```diff
@@ -103,11 +103,21 @@ jobs:
           ref: ${{ inputs.app_branch_name }}
           submodules: recursive
 
+      - name: Set binary path
+        run: |
+          if [[ "${{ needs.call_get_app_metadata.outputs.is_rust }}" == "true" ]];
+          then
+              echo "BINARY_PATH=${{ needs.call_get_app_metadata.outputs.build_directory }}/target/" >> $GITHUB_ENV
+          else
+              echo "BINARY_PATH=${{ needs.call_get_app_metadata.outputs.build_directory }}/build/" >> $GITHUB_ENV
+          if
+
       - name: Download app binaries
+        if: ${{ needs.call_get_app_metadata.outputs.is_rust == 'false' }}
```

Comment:
could be interesting to be able to get the binaries also for rust ?

Reply:
Oops, forgot to delete this line, thanks ;)